### DISCUSSION TITLE
Provides field descriptions for each report

### DISCUSSION THREAD

**Discussion by lanrenbulan** [amzn/selling-partner-api-models#591] [Ideas]
It would be better if the field description of each report can be provided, such as type, function, whether it is unique, etc.

At present, it is very inconvenient to know which fields the report has only after downloading the contents of the report document.

**Comment by MeenaAmz** [comment]
@lanrenbulan Can you please elaborate more on which report(s) you are referring to here and the difficulties you are facing?

**Reply by MeenaAmz** [reply]
[Report attributes](https://developer-docs.amazon.com/sp-api/docs/reports-attributes) has been added recently. Please check if this helps with your question.
